cli/config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
```

Comment:
Can config be made non-globals?

e.g. can it be created at the beginning and then passed around? (Makes DI easier in tests... ish)